fecgov/openFEC
# Feature/update locust and endpoints

## Summary (required)
update locust file and endpoints walk file
- Resolves #[_3618_]

_Include a summary of proposed changes._
- fixing some locust file data issues
- adding schedule_c/sub_id and schedule_d/sub_id to endpoints walking
## How to test the changes locally
1. test endpoints on prod
python endpoints_walk.py
note: the added endpoints walk will fail until the update is deployed.
2. run locust file against stage:
- check log file, make sure not 'more than 6 commitee' error from large_records_sched_a test
- make sure at least some test succeed with load_schedule_a_problematic test
-

## Impacted areas of the application
List general components of the application that this PR will affect:
api testing 
-  



## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
fix/other_pr | [link]()
feature/other_pr | [link]()
